styles/globals.css

```diff
@@ -14,3 +14,20 @@ a {
 * {
   box-sizing: border-box;
 }
+
+h3 {
+  font-size: 2em;
+  margin-bottom: 20px;
+}
+
+.MuiTypography-body1 {
+  margin-top: 16px;
+  margin-bottom: 16px;
+}
+
+.tool-box {
```

Comment:
For this class, since this is not really a global style that we want to apply to everything, it's better practice to put this in a css file specific for this page. You can add a home.css file in the styles folder, import it in _app.tsx and move the .tool-box style to home.css.